internal/system/agent/router.go

```diff
@@ -19,16 +19,29 @@ import (
 	"net/http"
 	"github.com/gorilla/mux"
 	"fmt"
+	"encoding/json"
+	"strings"
+	"github.com/edgexfoundry/edgex-go/pkg/models"
+)
+
+const (
+	edgexCoreCommand          = "edgex-core-command"
+	edgexCoreData             = "edgex-core-data"
+	edgexCoreMetadata         = "edgex-core-metadata"
+	edgexExportClient         = "edgex-export-client"
+	edgexExportDistro         = "edgex-export-distro"
+	edgexSupportLogging       = "edgex-support-logging"
+	edgexSupportNotifications = "edgex-support-notifications"
+	edgexConfigSeed           = "edgex-config-seed"
```

Comment:
All of these constants are already available here `internal/constants.go`. Please don't redefine.